client/analytics/settings/config.js

```diff
@@ -22,8 +22,9 @@ const DEFAULT_ORDER_STATUSES = [
 	'pending',
 	'on-hold',
 ];
+const DEFAULT_DATE_RANGE = 'period=month&compare=previous_year';
```

Comment:
This default was added by the client in `client/settings`, but I think it belongs here because its a default that does not come from the server and is only required here.